src/freenas/etc/directoryservice/LDAP/ctl

```diff
@@ -164,6 +198,11 @@ ldapctl_stop()
 		sssd_stop
 		ldapctl_cmd ${service} ix-sssd start
 	fi
+	if nslcd_running
+	then
+		nslcd_stop
+		ldapctl_cmd ${service} ix-nslcd start
```

Comment:
I couldn't find `ix-nslcd` in the master, does it exist or it should be `nslcd` here?

Reply:
Good question. I don't know! I'll look at the patch and see what I meant to do here.